File: backend/music/src/main/kotlin/at/htl/entity/Genre.java
package at.htl.entity;

import io.quarkus.hibernate.orm.panache.kotlin.PanacheEntity;

import javax.persistence.Entity;

@Entity
public class Genre extends PanacheEntity {
    public String name;
    public String description;

    public Genre() {
    }

    public Genre(String name, String description) {
        this.name = name;
        this.description = description;
    }
}

File: backend/music/src/main/kotlin/at/htl/entity/Genre.kt
package at.htl.entity

import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.Id

@Entity
class Genre() {

    @Id
    @GeneratedValue
    var id: Long? = null;
    lateinit var name: String
    lateinit var description: String

    constructor(name: String, description: String) : this() {
        this.name = name
        this.description = description
    }
}